tools/dockerfile/push_testing_images.sh

```diff
@@ -86,8 +86,8 @@ fi
 for DOCKERFILE_DIR in "${ALL_DOCKERFILE_DIRS[@]}"
 do
   # Generate image name based on Dockerfile checksum. That works well as long
-  # as can count on dockerfiles being written in a way that changing the logical 
```

Comment:
nit: I suggest reverting this file

Reply:
Done. Also, the test failures are all unrelated.